### PR TITLE
adding a demo gif, fixed expressions, badges, and add new badges to c…

### PR DESCRIPTION
**Pull request by victorvoid**
**CHANGES**:
- fixed a few sentences
- fixed badge of license, because it was using license badge of binaryage/chromex-sample
- Download link on top page and link to contribute
- Added a gif to show demonstration

**Warning** :warning: 
You need to fix your license, shield.io is not finding your license, try to set it manually.

:star2:

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,23 +1,46 @@
...
+  </h3>
+</div>
+</div>
+
+## How it works
+
+Just place the cursor over a specified function and see its spec. 
+![](demo.gif)
+
+You can see some fspec examples at [Bigmouth repo](https://github.com/athos/Bigmouth/blob/master/src/bigmouth/interaction.clj)).
```
Comment (pfeodrippe) [inline readme.md]: Just remove this last parenthesis and I'll merge it

**Comment by pfeodrippe** [comment]
@victorvoid Thanks for your time, Victor! It will look much more professional, gonna merge as soon as the requested fix is done o/

File: readme.md (hunk 1 above)
Comment (victorvoid) [reply readme.md]: done

**Comment by pfeodrippe** [comment]
@victorvoid Thank you very much!

**Pull request merged**